### DISCUSSION TITLE
DGP-848 PRT3

### DISCUSSION THREAD

**Discussion by 4D3K** [ParadoxAlarmInterface/pai#511] [General]
Can an older type of Paradox alarm system such as DGP-848 (Digiplex) be supported ? you can connect USB307 to it, but Babyware does not support it, only the older Winload software. 
I have a PRT3 module for it, which was advertised as a bridge to home automation, there is even a KNX module for it, but before I buy it I would like to know if maybe it is possible to communicate with Home Assistant through this particular module, using for example the built-in USB port. Is the protocol perhaps the same, or is it a completely different product ?

**Comment by yozik04** [comment]
No, not possible. Protocols are different. We can support only what Babyware can connect to. See [Compatibility page](https://github.com/ParadoxAlarmInterface/pai/wiki/Compatibility)
